src/main/java/ben/Events.java

```diff
@@ -0,0 +1,52 @@
+package ben;
+
+import java.time.LocalDateTime;
+
+/**
+ * Represents an events task.
+ */
+public class Events extends Task{
```

Comment:
Consulting the Java coding standards, perhaps using singular Event (without the s) is more accurate to represent a single Event instance. Events make it sounds like a collection class.

Reply:
Same for the ToDos and Deadlines!